File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/connection/AwsConnectionSettings.java
// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package software.aws.toolkits.jetbrains.ui.connection;

import javax.swing.JPanel;
import software.aws.toolkits.jetbrains.ui.CredentialProviderSelector;
import software.aws.toolkits.jetbrains.ui.RegionSelector;

public final class AwsConnectionSettings {
    JPanel panel;
    CredentialProviderSelector credentialProvider;
    RegionSelector region;
}

File: jetbrains-core/src/software/aws/toolkits/jetbrains/ui/connection/AwsConnectionSettings.kt
// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
package software.aws.toolkits.jetbrains.ui.connection

import software.aws.toolkits.jetbrains.ui.CredentialProviderSelector
import software.aws.toolkits.jetbrains.ui.RegionSelector
import javax.swing.JPanel

class AwsConnectionSettings {
    lateinit var panel: JPanel
    lateinit var credentialProvider: CredentialProviderSelector
    lateinit var region: RegionSelector
}
